internal/deploy/values/values.go

```diff
@@ -59,7 +59,8 @@ func addClusterSpecificDefaults(builder *builder, clusterInfo clusterinfo.Info)
 			addDefaultServerlessRegistryConfig(defaultRegistryConfig).
 			addDefaultGlobalDomainName(defaultLocalKymaDomain).
 			addDefaultGlobalTLSCrtAndKey(defaultLocalTLSCrtEnc, defaultLocalTLSKeyEnc).
-			addDefaultServerlessKanikoForce()
+			addDefaultServerlessKanikoForce().
+			addDefaultk3dValuesForIstio()
```

Comment:
This should be added only if the target cluster is K3d.

Reply:
```go
if k3d, isK3d := clusterInfo.(clusterinfo.K3d); isK3d {
```
This is not the correct place to have this param added?

Reply:
Ah yes, sorry... I missed the fact that this call is inside a block that only runs if the cluster is k3d.